doc/authoring_command_modules/authoring_commands.md

```diff
@@ -18,20 +18,24 @@ The document provides instructions and guidelines on how to author individual co
 
 [7. Supporting the IDs Parameter](#supporting-the-ids-parameter)
 
-[8. Generic Update Commands](#generic-update-commands)
 
-[9. Custom Table Formats](#custom-table-formats)
+[8. Supporting Name or ID Parameters](#supporting-name-or-id-parameters)
+
+[9. Generic Update Commands](#generic-update-commands)
 
 [10. Tab Completion (bash only)](#tab-completion)
 
-[11. Validators](#validators)
+[11. Custom Table Formats](#custom-table-formats)
+
+[12. Validators](#validators)
 
-[12. Registering Flag Arguments](#registering-flags)
 
 [13. Registering Enum Arguments](#registering-enums)
 
 [14. Preventing particular extensions from being loading](#extension-suppression)
 
+[15. Preventing particular extensions from being loading](#extension-suppression)
```

Comment:
This is duplicated.

Reply:
Yeah, I had to commit the merge and then fix it up.  Should be good now.